docfx/pulumi-template/layout/_master.tmpl

```diff
@@ -0,0 +1,55 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}

```

Comment:
If this is for our code - should we add our Copyright instead?

Reply:
It looks like we don't have copyright in our templates/styles in other parts of docs, so I just removed those top lines. Let me know if it's a wrong thing to do.